packages/ckeditor5-link/tests/utils/automaticdecorators.js

```diff
@@ -3,12 +3,12 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
  */
 
-import { AutomaticDecorators } from '../../src/utils/automaticdecorators.js';
+import { AutomaticLinkDecorators } from '../../src/utils/automaticdecorators.js';
```

Comment:
I believe the filename should reflect the exported member, which means it's a significant breaking change (for old installation methods). This needs to be confirmed, though.

Reply:
We kept old filenames globally, so this change is correct.